src/hotspot/cpu/x86/x86_32.ad

```diff
@@ -3712,6 +3712,7 @@ operand rRegP() %{
   interface(REG_INTER);
 %}
 
+// FIXME: The logic below do not apply anymore. Should we change anything? /ihse
```

Comment:
Here too we don't need Windows-specific support, since this is Linux only. But I need confirmation that the comment is correct so this code is really just Windows-specific.

Reply:
It looks like it is a dusty corner case. But the same logic as above applies: let's not touch it, and instead wait for it to go away with the remaining bits of 32-bit x86 port. I see `eRegP_no_EBP` is used for safepoint polls, so if we are wrong about the scope of this, rewriting these match rules to just `eRegP` might introduce surprising regressions.